docs/components.md

```diff
@@ -57,6 +57,24 @@ Setting `exposedEvents` to `true` rather than an array will expose all events.
 
 Additional events can also be exposed by passing in an array of event names to `exposedEvents` on the component options object.
 
+Events on components are namespaced when exposed. This means the events are bound to only the model of the component regardless of what level in the applicaton it is being used. The component's model events do not bubble up.
```

Comment:
Should we qualify "component's model events do not bubble up" with "unless they are exposed"?